packages/expo-image/ios/EXImage.m

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020-present 650 Industries. All rights reserved.
+
+#import <expo-image/EXImage.h>
```

Comment:
Oh, that's interesting — we've banned local imports since:
- they're clearer (explicitly showing the dependencies and where does the imported file come from)
- in the past they let us wrongly import files from the Xcode project in Pods
- we only support Cocoapods way of installing the unimodules (I think we should be safe to make this assumption in Expo modules as well since that's also where RN is headed, but I may be wrong on that)

Reply:
Oh that's interesting indeed. I've also experienced import path issues, but that was always due to a misconfiguration of some sorts (too many include/header paths in your XCode project config, causing header-hell). The new way of using auto-linking and specifying your deps in the Podfile seems to address this problem adequately.

It seems that pretty much all community packages are using local imports, and it seems to work just fine for regular (bare) projects (even the ones that are included in the Expo client).

Could it be that this is more related to versioning in the Expo client, and that it makes rewriting the imports easier? I did happen to find some projects (Yoga for instance) that also uses local imports and it seems to be versioned just fine.

This might be a matter of taste as well..

What exactly do you mean by?

> we only support Cocoapods way of installing the unimodules (I think we should be safe to make this assumption in Expo modules as well since that's also where RN is headed, but I may be wrong on that)

Reply:
Let's not let this be a blocker though. I'm okay with absolute imports 👍

Reply:
I think there are multiple ways of adding dependencies to Xcode projects, but two have emerged to be popular ones in RN:
- manually dragging a library's project ([Manual linking](https://reactnative.dev/docs/linking-libraries-ios#manual-linking)) — requires a library to have an `.xcodeproj` to drag 😃
- CocoaPods

For some reasons we have decided to use CocoaPods in Expo client and since most of the modules are developed this way we decided not to support the manual linking way of importing Expo-created modules. This is mostly dictated by no easy, automatic way to update `.xcodeproj` when a new file is added to the library (and CocoaPods handles this beautifully) and no workflow to test whether the `.xcodeproj` installation works.

For a moment we were thinking of writing something like a plugin to CocoaPods, based on `xcodeproj` gem that would autodetect all the `source_files` and add them to the `EXModule.xcodeproj`, but this never happened.

If I remember correctly:
- `""` imports are supported both when linking manually and when installing with CocoaPods since they are less restrictive
- `<>` imports are supported only when installing with CocoaPods

Since we're only supporting CocoaPods installations and `<>` are more restrictive, we went with the more restrictive version.

Maybe we should support manual linking in `expo-image`? If we decide to, we could do this in a separate PR that will also add the appropriate `.xcodeproj`.

Maybe we should revisit the CocoaPods plugin idea? `expotools update-vendored-module` is already able to modify the `Exponent.xcodeproj` when new files are added to vendored modules. Maybe it would be worth to copy this feature to some standalone library (if my memory doesn't fail me @tsapeta was the author of this part of code—what do you think)?

Another thing to consider (hats off to @tsapeta for reminding of this) is the (possible) ability to package modules as iOS frameworks (prebuilt libraries). If we achieved this, we could implement a performant Turtle service that  would allow building applications with/without modules (the process would only link libraries instead of building the whole project). I think importing files the `""` way would make it impossible. 🤔 

I agree, let's not bikeshed about this, I don't expect there to be many files in this library, so changing imports to `""` should be pretty easy.

I'm also curious what do other reviewers think. 🤔

Reply:
I've got no strong opinion on this matter.
Just a matter what's benefits us the most 😉